pkg/roll/update_test.go

```diff
@@ -683,8 +684,88 @@ func TestShouldRollMidwayHealthyMigrationFromZeroWhenNewSatisfies(t *testing.T)
 	upd.DesiredReplicas = 3
 	upd.MinimumReplicas = 2
 
-	remove, add, err := upd.shouldRollAfterDelay(rc_fields.RC{ID: upd.NewRC, Manifest: manifest})
+	remove, add, err := upd.shouldRollAfterDelay(manifest.ID())
 	Assert(t).IsNil(err, "expected no error determining nodes to roll")
 	Assert(t).AreEqual(remove, 0, "expected to remove no nodes")
 	Assert(t).AreEqual(add, 1, "expected to add one node")
 }
+
+func panicOnError(desc string, errs <-chan error) {
+	err, ok := <-errs
+	if ok {
+		panic("Error received on " + desc + ":" + err.Error())
+	}
+}
+
+// Transfers the named node from the old RC to the new RC
+func transferNode(node string, manifest pods.Manifest, upd update) error {
+	if _, err := upd.kps.SetPod(kp.REALITY_TREE, node, manifest); err != nil {
+		return err
+	}
+	podID := manifest.ID().String()
+	return upd.labeler.SetLabel(labels.POD, node+"/"+podID, rc.RCIDLabel, string(upd.NewRC))
```

Comment:
please remind me to use `MakePodLabelKey` which was added in https://github.com/square/p2/pull/452/files#diff-b534dcec1a1a54cde568a086e94fe0d3